tla/consensus/SIMccfraft.cfg

```diff
@@ -53,6 +53,11 @@ PROPERTIES
 POSTCONDITION
     WriteStatsFile
 
+\* ALIAS
```

Comment:
This needs to be uncommented to get the animation, presumably?

Reply:
Yes, you will pick the alias most suitable for the current debugging task.

Reply:
I tried it, it looks very cool!

Interestingly, I put a constraint on sentIndex to get it to fail, and it only ever (sample size ~12) generated single node configs and logs.

Reply:
I also observed less-than-ideal coverage, which is what I wanted to look at next.

Reply:
Yes, absolutely, this is very cool!